server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -339,6 +368,34 @@ const accountMutations = {
 
           break;
         }
+        case TwoFactorMethod.WEBAUTHN: {
+          const registrationResult = JSON.parse(Buffer.from(args.token, 'base64').toString('utf8'));
+          const registrationResponse = await webauthn.verifyRegistrationResponse(user, req, registrationResult);
+
+          if (!registrationResponse.verified) {
```

Comment:
Wouldn't `webauthn.verifyRegistrationResponse` throw before you reach this conditional?

Reply:
I think so, but not sure. I think I am being defensive here because this might depend on the type of device used: https://github.com/MasterKale/SimpleWebAuthn/blob/7cbc1dc6723294874fe4f576c2d32d07337ec5af/packages/server/src/registration/verifyRegistrationResponse.ts#L208]

We are requesting registration with `attestation = 'direct'`, so it should always be verifiable.

I am thinking about just moving this check to our wrapper, instead of here in the mutation, wdyt?